lms/envs/production.py

```diff
@@ -1133,6 +1133,8 @@ def get_env_setting(setting):
 
 SURVEY_REPORT_CHECK_THRESHOLD = ENV_TOKENS.get('SURVEY_REPORT_CHECK_THRESHOLD', 6)
 
+ENABLE_SURVEY_REPORT = ENV_TOKENS.get('ENABLE_SURVEY_REPORT', True)
```

Comment:
This should be set in common.py and not in production.py  Defaults should not live in production.py in general and should be pulled from common.py.  I see the setting above has the same issue but I don't want to add more settings that are using the wrong pattern.

Reply:
Hello @feanil should we move all the settings we have for survey report into common.py? Or just the 2 settings mentioned above?

Reply:
Since we are fixing this for good I think all of them. It can be in a separate PR if it makes it more orderly

Reply:
I have moved all survey report settings to common.py, let me know if we need something more. @feanil @felipemontoya